include/watchdog.h

```diff
@@ -19,11 +19,11 @@
  * Public API Function to enable watchdog.
  * Must be called in initializeIO
  */
-void iwdgEnable();
+void watchdogEnable();
```

Comment:
Is this fine as `watchdogEnable` as opposed to `watchdogInit` (which conforms to the naming standards a little better)?